recipes/tensorflow-probability/0001-always-build-release.patch

```diff
@@ -0,0 +1,15 @@
+diff --git a/setup.py b/setup.py
+index 5a43400..2208279 100644
+--- a/setup.py
++++ b/setup.py
+@@ -33,9 +33,8 @@ REQUIRED_PACKAGES = [
+     'numpy >= 1.13.3',
+ ]
+ 
+-if '--release' in sys.argv:
++if True:
+   release = True
+-  sys.argv.remove('--release')
+ else:
+   # Build a nightly package by default.
+   release = False
```

Comment:
Pip has an option to pass flags like this. I forget the name. I’m ok with the patch but it might be more maintainable in the future

Reply:
It does, it is either --install-options or --global-options. I used that before but there were still messages on the screen indicating that it was building the nightly package instead of the release.

I opted to reduce the chance of installing the wrong package name.

Reply:
Ahhhh ok. Good call!

Reply:
It might be good I leave a comment with the info above but I’m not too worried.